python/valis/routes/target.py

```diff
@@ -195,6 +196,26 @@ async def get_catalogs(self, sdss_id: int = Path(title="The sdss_id of the targe
 
         return response_data
 
+    @router.get('/parents/{parent_catalog}/{sdss_id}',
+                dependencies=[Depends(get_pw_db)],
+                response_model=ParentCatalogModel,
+                summary='Retrieve parent catalog information for a taget by sdss_id')
+    async def get_parents(self,
+                          parent_catalog: str = Path(title='The parent catalog to search',
+                                                     example='gaia_dr3_source'),
+                          sdss_id = Path(title='The sdss_id of the target to get',
+                                         example=129055990)):
+        """Return parent catalog information for a given sdss_id """
+
+        try:
+            result = get_parent_catalog_data(sdss_id, parent_catalog).dicts()
+            if len(result) == 0:
+                raise ValueError(f'No parent catalog data found for sdss_id {sdss_id}')
+        except Exception as e:
+            raise HTTPException(status_code=500, detail=f'Error: {e}')
```

Comment:
I wonder if this should be a 400 error instead, which is `Bad Request`?  I feel like in this case, the error is a known error caused by the user submitting an incorrect catalog for the given sdssid, rather than an unknown server error.

Reply:
I guess this is a user/client error so 400 sounds reasonable.